spring-integration-amqp/src/main/java/org/springframework/integration/amqp/inbound/AmqpInboundGateway.java

```diff
@@ -132,26 +151,155 @@ public void setDefaultReplyTo(String defaultReplyTo) {
 		this.defaultReplyTo = new Address(defaultReplyTo);
 	}
 
+	/**
+	 * Set a {@link RetryTemplate} to use for retrying a message delivery within the
+	 * gateway. Unlike adding retry at the container level, this can be used with an
+	 * {@code ErrorMessageSendingRecoverer} {@link RecoveryCallback} to publish to the
+	 * error channel after retries are exhausted. You generally should not configure an
+	 * error channel when using retry here, use a {@link RecoveryCallback} instead.
+	 * @param retryTemplate the template.
+	 * @see #setRecoveryCallback(RecoveryCallback)
+	 * @since 4.3.10.
```

Comment:
The `@see` must be after `@since`:
```
<property name="tagOrder" value="@param, @return, @throws, @since, @deprecated, @see"/>
```
and https://github.com/spring-projects/spring-framework/wiki/Spring-Framework-Code-Style#javadoc-formatting

I wish we switch the rule on one day in the future 😄